drivers/drv8825/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! DRV8825 Driver
+//!
+//! Platform-agnostic driver library for the DRV8825 stepper motor driver.
+//! This crate is a specialized facade of the [Step/Dir] library. Please
+//! consider using Step/Dir directly, as it provides drivers for more stepper
+//! motor drivers, as well as an interface to abstract over them.
+//!
+//! See [Step/Dir] for more documentation and usage examples.
+//!
+//! [Step/Dir]: https://crates.io/crates/step-dir
+
+#![no_std]
+#![deny(missing_docs)]
+
+pub use step_dir::*;
+
+mod drv8825;
+pub use drv8825::*;
```

Comment:
Can you please move the `drv8825` module to `step-dir` (behind a feature flag), like the STSPIN220 driver does it? I want to have separate crates for convenience and searchability, but I think some use cases are better served by a unified crate. With re-exports and feature flags, we can have the best of both worlds.